### ISSUE TITLE
Rancher v0.18.2 on Fedora 21 broken

### ISSUE THREAD

**Issue by cloudnautique** [rancher/rancher#665]
Environment:
Docker 1.6 
Fedora 21 (GCE)

Rancher agent v0.6.1
This error shows up in the agent logs:

```
2015-05-01 04:25:00,958 ERROR agent [140118420566256] [event.py:108] 3aaa7e79-a0a9-4c5f-a59f-cc2023455bd4 : Unknown error
Traceback (most recent call last):
  File "/var/lib/cattle/pyagent/cattle/agent/event.py", line 91, in _worker_main
    resp = agent.execute(req)
  File "/var/lib/cattle/pyagent/cattle/agent/__init__.py", line 15, in execute
    return self._router.route(req)
  File "/var/lib/cattle/pyagent/cattle/plugins/core/event_router.py", line 13, in route
    resp = handler.execute(req)
  File "/var/lib/cattle/pyagent/cattle/agent/handler.py", line 33, in execute
    return method(req=req, **req.data.__dict__)
  File "/var/lib/cattle/pyagent/cattle/storage/__init__.py", line 23, in image_activate
    action=lambda: self._do_image_activate(image, storage_pool,
  File "/var/lib/cattle/pyagent/cattle/agent/handler.py", line 71, in _do
    action()
  File "/var/lib/cattle/pyagent/cattle/storage/__init__.py", line 24, in <lambda>
    progress)
  File "/var/lib/cattle/pyagent/cattle/plugins/docker/storage.py", line 86, in _do_image_activate
    message = status['error']
  File "/var/lib/cattle/pyagent/cattle/utils.py", line 51, in __getitem__
    value = self.__dict__[item]
KeyError: 'error'
```

**Comment by Trisooma** [comment]
I have the same issue:
**docker version**
```
Client version: 1.6.0
Client API version: 1.18
Go version (client): go1.4.2
Git commit (client): 4749651
OS/Arch (client): linux/amd64
Server version: 1.8.1.fc22
Server API version: 1.20
Go version (server): go1.4.2
Git commit (server): 32b8b25/1.8.1
OS/Arch (server): linux/amd64
```
**docker info**
```
Containers: 8
Images: 84
Storage Driver: devicemapper
 Pool Name: docker-253:0-2232088-pool
 Pool Blocksize: 65.54 kB
 Backing Filesystem: extfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 4.626 GB
 Data Space Total: 107.4 GB
 Data Space Available: 25.7 GB
 Metadata Space Used: 5.476 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.142 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Data loop file: /data/VirtImage/docker/devicemapper/devicemapper/data
 Metadata loop file: /data/VirtImage/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.93 (2015-01-30)
Execution Driver: native-0.2
Kernel Version: 4.1.4-200.fc22.x86_64
Operating System: Fedora 22 (Twenty Two)
CPUs: 4
Total Memory: 15.6 GiB
Name: ************
ID: H27H:5UJ2:UU5J:UNCD:JPDP:QHZ7:PWTR:44YS:XDOU:E22L:LQJ5:LNOX
Http Proxy: 
Https Proxy: 
No Proxy: 
```
**docker agent v0.8.2**

**Comment by deniseschannon** [comment]
Closing as Fedora is on 23
